ui/src/dashboards/components/dashboard_empty/DashboardEmpty.tsx

```diff
@@ -30,13 +25,21 @@ class DashboardEmpty extends Component<Props> {
             size={ComponentSize.Medium}
             icon={IconFont.AddCell}
             color={ComponentColor.Primary}
-            onClick={onAddCell}
+            onClick={this.handleAdd}
             testID="add-cell--button"
           />
         </EmptyState>
       </div>
     )
   }
+
+  private handleAdd = () => {
+    // TODO(alex): change this to using state from redux
```

Comment:
Let's do this now!  You should already have the `dashboardID` and `orgID` from the `WithRouterProps`